lib/compile.js

```diff
@@ -130,7 +131,7 @@ var Compiler = (function () {
                     msgid = msgid.replace( unconvertedEntityPattern, convertedEntity );
                 }
 
-                if ( removeWhiteSpaces ) {
+                if ( removeWhiteSpaces && !isDBString ) {
```

Comment:
What's with the weird spaces between the parentheses? That's jQuery's way of doing JavaScript but we definitely don't do it.